src/revocation.rs

```diff
@@ -274,4 +309,51 @@ mod tests {
         let outputs = run_revocation_actions(json, &test_config.secure_size);
         assert!(outputs.is_err());
     }
+
+    #[test]
+    fn get_revocation_cert_path_default() {
+        let test_config = KeylimeConfig::default();
+        let revocation_cert_path =
+            get_revocation_cert_path(&test_config).unwrap();
```

Comment:
Looks like the CI is failing on a few of these `unwrap()`s still.

Reply:
My bad, I'll fix this